iml-gui/crate/src/page/partial/header.rs

```diff
@@ -282,7 +282,10 @@ pub fn auth_view(auth: &auth::Model, logging_out: bool) -> Node<Msg> {
                     attrs! {
                         At::Href => Route::User(user.id.into()).to_href()
                     },
-                    user.username
+                    match (user.full_name.as_str(), user.username.as_str()) {
+                        ("", x) => x,
```

Comment:
Basically, `if user.full_name.is_empty()...`?

Reply:
yeah. 6 of one half-dozen of the other